wiki/Tournaments/NSS/1/en.md

```diff
@@ -0,0 +1,419 @@
+6. There is one protect in all rounds. Protected maps cannot be banned.
+7. There are no restrictions on picks and bans (double banning and triple/double picking are allowed).
+8. If there are multiple bans, they are in ABAB order, A being the "first ban".
+9. After bans, maps are picked in alternating order starting with the first picker.
+10. At match point (when both players are one point from winning), the Tiebreaker will be played.
+11. Players have 90 seconds to pick maps and 90 seconds to ready up.
+12. Each player has **TWO timeouts** lasting 90 seconds each, which are used to extend timers. These can be taken at any point in the match after the first map is played, and can be taken consecutively.
+    - Referees may force timeouts if players do not act in time, even if the first map has not yet been played.
+13. If a player allows a timer to expire and has no timeouts left:
+    - Pick, ban, or protect: The referee !rolls the amount of possible maps and uses the result as the player’s choice.
+    - Ready-up timer: !mp start 15 is forced.
+14. For FM maps, mods are optional, but NM = 0.8x, EZ, EZHD, or EZFL = 1.7x, and FL = 1.4x.
+15. If both players somehow get *exactly* the same score, the player with better accuracy wins. If there is still a tie, the map is replayed.
+    - Mods can be changed if a map is replayed for this reason.
+16. In case of a disconnect:
+    - The first time, the map is replayed if the disconnect happened within the first **30 seconds.**
+    - If the same player disconnects again, they lose unless they can prove their score, for example, by Shift+F12 or replay file.
+17. If players do not join the match at the given time, the following penalties apply:
+    - 5 minutes: No warmup
+    - 8 minutes: No protect
+    - 10 minutes: No bans
+    - 15 minutes: Forfeit
+18. If a player becomes silenced during the match:
+    - 5 minutes: Timeouts lost or next pick/ban lost
+    - 10 minutes: Timeouts lost **and** next pick/ban lost
+    - 20 minutes or more: **Forfeit** unless the opponent wants to reschedule
+19. If a player becomes silenced while playing a map, it counts as a disconnect.
+
+### Schedule Rules
+
+1. Qualifiers lobbies should be scheduled at least **6 hours** in advance.
+2. In qualifiers, players may schedule an EX lobby at any time. Depending on referee availability, this may not be granted.
+3. Matches should be scheduled at least **24 hours** in advance.
+4. Reschedule requests should be considered reasonably by the receiving player. If a player appears to be denying reschedule requests unfairly, the administration will intervene.
+5. Reschedule requests with faked confirmation from the other player will result in a forfeit.
+
+### Group Stage
+
+1. During the Group Stage, there are 3 to 5 players in each group, depending on registrations.
+2. These players all play 1 match against each other over two weeks.
```

Comment:
bit weird to mix these in the same sentence

pedantically, small numbers in sentences should *either* be spelled out or written in numerals, for the whole article

```suggestion
2. These players all play one match against each other over two weeks.
```